corehq/apps/reports/tasks.py

```diff
@@ -260,3 +273,142 @@ def _store_excel_in_redis(file):
     r.expire(hash_id, EXPIRE_TIME)
 
     return hash_id
+
+
+@task
+def build_form_multimedia_zip(domain, xmlns, startdate, enddate, app_id, export_id, zip_name, download_id):
+
+    def find_question_id(form, value):
```

Comment:
I think you can delete this from views.py since it doesn't seem to be used anywhere else.